@ndlib/gatsby-theme-marble/src/components/Internal/Seo/SeoContent/index.js

```diff
@@ -17,10 +18,11 @@ export const SeoContent = ({
   siteUrl,
   noIndex,
 }) => {
-  const openGraph = getOpenGraph(title, description, image)
+  const openGraph = getOpenGraph(url, title, description, image)
   const twitter = getTwitter(author, title, description, image)
   const titleFix = title.includes('Mirador Viewer') ? title : `${title} | ${siteTitle}`
   // console.log('=======================================================')
```

Comment:
I would like to see all these comments removed.  We can always re-add tests to the code base that we want to debug things later.

Reply:
Updated unit test and removed comments. Also fixed `getUrl()`:
```
export const getUrl = (siteMetadata, location) => {
  return typy(siteMetadata, 'siteUrl').safeString + location
}
```